js/tests/unit/button.js

```diff
@@ -60,6 +60,19 @@ $(function () {
     assert.strictEqual($btn.attr('aria-pressed'), 'true', 'btn aria-pressed state is true')
   })
 
+  QUnit.test('should toggle aria-pressed on buttons with container', function (assert) {
```

Comment:
@bardiharborow should we add ``qunit`` in ``.eslintrc.json`` to avoid this ? 
see : http://eslint.org/docs/user-guide/configuring#specifying-environments